### PR TITLE
Switching windows builds back to appveyor

### PR DESCRIPTION
**Pull request by smmaurer**
This PR is to check whether we can continue building for Python 2.7 on Windows by switching back from Azure to AppVeyor. If this causes headaches, i'll just delete it without merging.

(We surveyed some users and a fair number are still using win-py27, unfortunately.)

Some relevant links: 
- this is the commit that disabled win-py27 builds, i think: [78d29b2](https://github.com/conda-forge/pandana-feedstock/pull/1/commits/78d29b22a170ef93b0bee04a2644499cb8ab9d3e)
- and here is some related documentation: 
   https://conda-forge.org/docs/maintainer/infrastructure.html#using-azure-for-everything

Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by smmaurer** [comment]
@conda-forge-admin, please rerender

File: conda-forge.yml [hunk 1]
```diff
@@ -4,5 +4,3 @@ travis:
 appveyor:
   secure:
     BINSTAR_TOKEN: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1
-provider:
-  win: azure
```
Comment (ocefpaf) [inline conda-forge.yml:8]: The failure on Python 2.7 is related to vc9 and not AppVeyor vs Azure. Let's keeo Azure here b/c it is faster and we are slowly moving away from AppVeyor.

**Comment by smmaurer** [comment]
It looks like the failure is probably from not including `#include <functional>` in `accessibility.cpp`, which i guess is required by certain compilers:

```
src/accessibility.cpp(353) : error C2039: 'function' : is not a member of 'std'
```

We can fix it in the source code for the next release.

File: conda-forge.yml (hunk 1 above)
Comment (smmaurer) [reply conda-forge.yml:8]: Ok, got it. From the documentation page it looked like Azure can't do Python 2.7 builds for Windows at all, but i see there's an error in AppVeyor too. I'll close this without merging, and try again when we have a new release!

https://conda-forge.org/docs/maintainer/infrastructure.html#using-azure-for-everything

Comment (ocefpaf) [reply conda-forge.yml:8]: You are right. Some things may fail. We can re-enable AppVeyor if that is the case.

**Pull request closed without merging**